### DISCUSSION TITLE
Hearing aids

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#394] [General]
We are looking for a good pair of hearing aids for my dad as his hearing got much worse over the last two years. Any recommendations? Something which is good quality and doesn't cost a fortune?
Appreciate your feedback.

**Comment by ellielo** [comment]
So far i didn't find any program which will pay for hearing aids. Would be just nice to know if there is a point to spend $5000-$7000 for aids or those for $1000-$2000 can still do a good job. The variety of hearing aids is just insane. I am going to [staten island audiological ](https://audiologyisland.com/)in two weeks and hope they can give me some good advices.
